vspc/server.py

```diff
@@ -195,6 +198,24 @@ async def save_to_log(self, uuid, data):
         fpath = os.path.join(CONF.serial_log_dir, uuid)
         async with aiofiles.open(fpath, 'ab') as f:
             await f.write(data)
+        if CONF.log_file_size and os.path.getsize(fpath) > CONF.log_file_size:
+            truncated_data = await self.truncate_log_file(fpath, CONF.log_file_size)
+            await self.rewrite_file(fpath, truncated_data)
+
+    async def truncate_log_file(self, file_name, size):
+        async with aiofiles.open(file_name, "rb") as read_obj:
+            await read_obj.seek(0, os.SEEK_END)
+            pointer_location = await read_obj.tell()
+            start_point = pointer_location - size
+            await read_obj.seek(start_point)
+            return await read_obj.read()
```

Comment:
The logs are line based, so I'd prefer not having half lines in there because of seeking. What do you think? Would it make sense to go further until the next line? (Maybe it's as easy as creating a file from it and doing a `readline()` before the final read.)

Reply:
it was the first solution, but when you said that we don't care about past old logs I've decided just to seek to point of bytes. :) And yes, it's better when we have a whole line rather than a broken one...